nosekatreport/test_aqf.py

```diff
@@ -62,23 +69,27 @@ def test_aqf_decorators_2(self):
         Aqf.is_false(status, "Check that the sensor status is now false")
         Aqf.end()
 
-    @intrusive
-    @aqf_vr('VR.CM.EXAMPLE.17')
-    def test_aqf_decorators_3(self):
```

Comment:
Why is this test now disabled?  If we don't need it, then it should be removed completely.  If we do need it, we should rather get it working.

Reply:
Well, CBF doesn't use the decorator hence I added comment, for CAM-ers to add working code if needed, else we can delete the method.

Reply:
@lvdheever Do you expect this test to be working?

Reply:
I haven't run nosekatreport tests myself for ages, but I suspect the problem here is not the @intrusive decorator but rather passing a string that cannot translate to a KATCP sensor to AQF: Aqf.sensor('sim.sensors.asc_wind_speed').get().
I expect Aqf.sensor(xxx).get() to be working, but the test seems to be wrong as it is not mocking out enough.

Reply:
@lvdheever, do we comment out the test and merge or someone is going to try to fix the issue?

Reply:
I suggest we uncomment the test, but add the `@unittest.skip('CAM to implement')` decorator.  Then things are clear.

Reply:
CAM won't have time to look into this now. Mpho, if you do not have time to investigate and fix the test (it is a core test to AQF and not CAM related) then I will go with Anton's suggestion as 2nd prize.

Reply:
@lvdheever I've implemented @ajoubertza suggestion, will commit and push